server/src/models/job_posts.model.js

```diff
@@ -0,0 +1,96 @@
+require("dotenv").config();
+const { DataTypes } = require("sequelize");
+const { sequelize } = require("../configs/sequelize");
+const User = require("./user.model");
+
+const JobPosting = sequelize.define(
+  "job_postings",
+  {
+    id: {
+      type: DataTypes.INTEGER,
+      primaryKey: true,
+      autoIncrement: true,
+    },
+    title: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    employer: {
+      type: DataTypes.STRING,
+      allowNull: false,
```

Comment:
Apologies for the late update, but we need to change some of the allowNull properties on these fields to allow users to save half-completed job applications. I added a description of this to the ticket on Linear

Reply:
The latest commit includes this change